core/metacling/src/TClingClassInfo.cxx

```diff
@@ -634,6 +634,33 @@ ptrdiff_t TClingClassInfo::GetBaseOffset(TClingClassInfo* base, void* address, b
    return binfo.Offset(address, isDerivedObject);
 }
 
+std::vector<std::string> TClingClassInfo::GetUsingNamespaces()
+{
+   // Find and return all 'using' declarations of namespaces.
+   std::vector<std::string> res;
+
+   R__LOCKGUARD(gInterpreterMutex);
+
```

Comment:
```suggestion
if (!isa<DeclContext>(DC))
  return {};
```

Reply:
We have a cast right after, and that should be a `dyn_cast` - see my comment below for the more common pattern.